app/assets/stylesheets/passengers.scss

```diff
@@ -115,6 +115,6 @@ div {
   }
 }
 
-.hide_view {
+.hide-view {
```

Comment:
FWIW, Bootstrap has `.hidden` already.

Reply:
I was looking for that!!

Reply:
OK, I tried this out--doing this makes Jquery's `toggle()` not work, because of the `!important` aspect of the `display: none` property of the lass `.hidden`, which, you pointed out, it still somehow more important than inline styles.